### DISCUSSION TITLE
pyshark method to get entire packet info

### DISCUSSION THREAD

**Discussion by sajumadnair** [KimiNewt/pyshark#599] [Q&A]
Is there a simple method to access the entire packet hex value , including all layers.
I am interested in the red marked section fully

![Capture](https://user-images.githubusercontent.com/24620158/190156187-fbe0fc29-e857-4a32-b9b9-c0bfa8c61c67.PNG)

**Comment by fosterwolff** [comment]
capture = pyshark.LiveCapture(interface='Ethernet',use_json=True, include_raw=True)
capture.sniff(timeout=2)
for packet in capture:
    input(packet)
